src/llmq/init.cpp

```diff
@@ -16,25 +16,27 @@
 #include <llmq/utils.h>
 #include <consensus/validation.h>
 
+#include <node/context.h>
+
 #include <dbwrapper.h>
 
 namespace llmq
 {
 
 CBLSWorker* blsWorker;
 
-void InitLLMQSystem(CEvoDB& evoDb, CTxMemPool& mempool, CConnman& connman, bool unitTests, bool fWipe)
+void InitLLMQSystem(NodeContext& node, CEvoDB& evoDb, CTxMemPool& mempool, CConnman& connman, bool unitTests, bool fWipe)
 {
     blsWorker = new CBLSWorker();
 
-    quorumDKGDebugManager = new CDKGDebugManager();
+    node.quorumDKGDebugManager = std::make_unique<CDKGDebugManager>();
```

Comment:
Consider using `make_shared`/`make_shared` and migrate all `new` instantiations to it as a separate commit. Picking between either seems to involve a tradeoff between safety and performance. Thoughts @UdjinM6?

> The C++17 standard, while still not specifying the exact evaluation order for arguments, provides additional guarantees. All side effects of a function argument must be evaluated before the next argument is evaluated. This helps mitigate the risk in case of exceptions, but it is still preferable to use 'make_unique'.
>
> One thing should be noted about the 'make_shared' function. When it is used, the pointer's control block is allocated next to the managed object. This helps reduce the number of dynamic allocations and optimize the use of the CPU cache.

Source: https://pvs-studio.com/en/docs/warnings/v824/